doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -537,67 +479,76 @@ Linear Algebra Equivalents
      - ``U, S, Vh = linalg.svd(a), V = Vh.T``
      - singular value decomposition of ``a``
 
-   * - ``chol(a)``
-     - ``linalg.cholesky(a).T``
+   * - ``c=chol(a)`` where ``a==c'*c``
+     - ``c = linalg.cholesky(a)`` where ``a == c@c.T``
      - cholesky factorization of a matrix (``chol(a)`` in MATLAB returns an
        upper triangular matrix, but ``linalg.cholesky(a)`` returns a lower
        triangular matrix)
 
    * - ``[V,D]=eig(a)``
      - ``D,V = linalg.eig(a)``
-     - eigenvalues and eigenvectors of ``a``
+     - eigenvalues :math:`\lambda` and eigenvectors :math:`\bar{v}` of ``a``,
+       where :math:`\lambda\bar{v}=\mathbf{a}\bar{v}`
 
    * - ``[V,D]=eig(a,b)``
-     - ``D,V = scipy.linalg.eig(a,b)``
-     - eigenvalues and eigenvectors of ``a``, ``b``
+     - ``D,V = linalg.eig(a,b)``
+     - eigenvalues :math:`\lambda` and eigenvectors :math:`\bar{v}` of
+       ``a``, ``b``
+       where :math:`\lambda\mathbf{b}\bar{v}=\mathbf{a}\bar{v}`
 
-   * - ``[V,D]=eigs(a,k)``
-     -
-     - find the ``k`` largest eigenvalues and eigenvectors of ``a``
+   * - ``[V,D]=eigs(a,3)``
+     - ``D,V = eigs(a,k=3)``
+     - find the ``k=3`` largest eigenvalues and eigenvectors of 2D array, ``a``
 
    * - ``[Q,R,P]=qr(a,0)``
-     - ``Q,R = scipy.linalg.qr(a)``
+     - ``Q,R = linalg.qr(a)``
      - QR decomposition
 
-   * - ``[L,U,P]=lu(a)``
-     - ``L,U = scipy.linalg.lu(a)`` or ``LU,P=scipy.linalg.lu_factor(a)``
-     - LU decomposition (note: P(MATLAB) == transpose(P(numpy)) )
+   * - ``[L,U,P]=lu(a)`` where ``a==P'*L*U``
+     - ``P,L,U = linalg.lu(a)`` where ``a==P@L@U``
+     - LU decomposition (note: P(MATLAB) == transpose(P(NumPy)))
 
    * - ``conjgrad``
-     - ``scipy.sparse.linalg.cg``
+     - ``cg``
      - Conjugate gradients solver
 
    * - ``fft(a)``
-     - ``fft(a)``
+     - ``np.fft(a)``
      - Fourier transform of ``a``
 
    * - ``ifft(a)``
-     - ``ifft(a)``
+     - ``np.ifft(a)``
      - inverse Fourier transform of ``a``
 
    * - ``sort(a)``
-     - ``sort(a)`` or ``a.sort()``
-     - sort the matrix
+     - ``np.sort(a)`` or ``a.sort(axis=0)``
+     - sort each column of a 2D matrix, ``a``
+
+   * - ``sort(a,2)``
+     - ``np.sort(a,axis=1)`` or ``a.sort(axis=1)``
+     - sort the each row of 2D matrix, ``a``
 
-   * - ``[b,I] = sortrows(a,i)``
-     - ``I = argsort(a[:,i]), b=a[I,:]``
-     - sort the rows of the matrix
+   * - ``[b,I]=sortrows(a,1)``
+     - ``I=np.argsort(a[:,0]); b=a[I,:]``
+     - save the array ``a`` as array ``b`` with rows sorted by the first column
 
-   * - ``regress(y,X)``
-     - ``linalg.lstsq(X,y)``
-     - multilinear regression
+   * - ``Z\y``
+     - ``linalg.lstsq(Z,y)``
+     - perform a linear regression of the form :math:`\mathbf{Zx}=\mathbf{y}`
```

Comment:
```suggestion
   * - ``x = Z\y``
     - ``x = linalg.lstsq(Z,y)``
     - perform a linear regression of the form :math:`\mathbf{Zx}=\mathbf{y}`
```
otherwise it's not clear what `x` is

Reply:
Or maybe
```suggestion
   * - ``Z\y``
     - ``linalg.lstsq(Z,y)``
     - the solution :math:`x` to a linear regression of the form :math:`\mathbf{Zx}=\mathbf{y}`
```